static/locales/en-US.yaml

```diff
@@ -241,6 +241,8 @@ User Playlists:
       Playlist {playlistName} is the new quick bookmark playlist.: Playlist {playlistName} is the new quick bookmark playlist.
 
       This playlist does not exist: This playlist does not exist
+
+      This playlist has a video with a duration error: This playlist contains at least one video that doesn't have a duration, it will be sorted as if their duration is zero.
```

Comment:
**question:** Just for discussion, thoughts on moving videos with no duration to the bottom of the stack (i.e., `Number.MAX_SAFE_INTEGER` when sort ascending, `0` when sort descending)? Would result in fewer complaints, and would be preferable if for some reason a playlist had multiple of these consistently because they wouldn't have to be scrolled past each time